### ISSUE TITLE
connection.LoadAsync<TEntity>  throws

### ISSUE THREAD

**Issue by AsafMag** [danikf/tik4net#38]
The following code:
```
...
connection.LoadAsync<WirelessAccessList>(
    (item) => { },
    (ex) => { }
);
...
```

Throws:
```
interface/wireless/access-list
MESSAGE: no such command
```

Am I using it wrong?

**Comment by danikf** [comment]
Hi,
I don't understand the reason why load list (sample command returns list just once) with async method. It didn't work (LoadAsync didn't add /print suffix). This behavior has been fixed by 843b50ab7323d8a7227326b4ebf6ecf835d52a94 commit.

So - with DEV tik4net version your sample code will work correctly (but synchronous load is still recomended).

D

**Comment by AsafMag** [comment]
It would actually be great if you added a tag for the synchronous commands as well, this would make it a lot easier for me. This is the reason I was trying to use `LoadAsync`
